packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.test.tsx

```diff
@@ -69,34 +83,15 @@ describe('PageAccountRecoveryConfirmKey', () => {
   //   bundle = await getFtlBundle('settings');
   // });
 
-  beforeEach(() => {
-    mockCode = 'code';
-    mockToken = 'token';
-    mockEmail = 'boo@boo.boo';
-    mockUid = 'uid';
-
-    account = {
-      getRecoveryKeyBundle: jest.fn().mockResolvedValue({
-        recoveryData: 'mockRecoveryData',
-        recoveryKeyId: MOCK_RECOVERY_KEY_ID,
-      }),
-      verifyPasswordForgotToken: jest
-        .fn()
-        .mockResolvedValue({ accountResetToken: MOCK_RESET_TOKEN }),
-    } as unknown as Account;
-  });
-
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it('renders as expected when the link is valid', () => {
-    render(<Subject {...{ account }} />);
+  it('renders as expected when the link is valid', async () => {
+    renderSubject(accountWithValidResetToken);
```

Comment:
Since `accountWithValidResetToken` is pretty commonly used, you could set it as a default value in the function so you don't have to pass it in so often.